src/main/java/net/mcreator/ui/dialogs/preferences/ThemesPanel.java

```diff
@@ -88,11 +88,11 @@ public Component getListCellRendererComponent(JList<? extends Theme> list, Theme
 			String text = "<html>" + value.getName();
 			if (!value.getDescription().isEmpty())
 				text += "<br><small>" + value.getDescription();
-			text += "<br>Theme ID: " + value.getID();
+			text += L10N.t("dialog.preferences.id") + value.getID();

```

Comment:
Same here, better to leave untranslated as other languages can have different order here, but you can't use replacement patterns due to conditional nature of these fields.